src/test/java/plugins/WorkflowPluginTest.java

```diff
@@ -158,4 +168,31 @@
         build.shouldContainsConsoleOutput("divided into 3 sets");
     }
 
+    @WithDocker
+    @WithPlugins({"workflow-aggregator@1.14", "docker-workflow@1.14-SNAPSHOT", "git", "ssh-agent@1.10-SNAPSHOT"})
```

Comment:
Downstream of https://github.com/jenkinsci/docker-workflow-plugin/pull/33 and https://github.com/jenkinsci/ssh-agent-plugin/pull/11.